package.json

```diff
@@ -4,25 +4,21 @@
   "repository": {},
   "license": "MIT",
   "devDependencies": {
-    "babel-core": "^6.24.1",
+    "babel-core": "^6.26.0",
     "babel-loader": "^7.0.0",
-    "babel-plugin-module-resolver": "^2.7.1",
-    "babel-polyfill": "^6.13.0",
```

Comment:
Have you tested this on older browsers? Removing polyfill will likely break stuff on IE.

Reply:
@ryanlntn at this time we only officially support Chrome for editing. Down the road, we should support all latest browsers. Speaking of, there needs to be some sort of messaging that conveys this to the user. I can work on that.

Reply:
IE... ![https://images.duckduckgo.com/iu/?u=http%3A%2F%2Fimg.memecdn.com%2Fwhen-i-try-not-to-laugh_o_468631.jpg&f=1](https://images.duckduckgo.com/iu/?u=http%3A%2F%2Fimg.memecdn.com%2Fwhen-i-try-not-to-laugh_o_468631.jpg&f=1)

Reply:
It should be pretty possible with the new `babel-preset-env` to support whatever we want. It polyfills back as far as you want it to.